src/components/LayoutColumns/index.js

```diff
@@ -7,6 +7,7 @@ export default ({items, relativeSizes}) => (
       <div
         key={index}
         className={`
+          mb4
```

Comment:
Layout columns take care of gutters by device size (horizontal and vertical) so components don't need wrapping padding/margin